src/renderer.ts

```diff
@@ -329,17 +323,17 @@ const richLinkStyles = css`
     ${darkModeCss`
         background-color: ${neutral[20]};
         color: ${neutral[60]};
-        a {
+
+        h1 {
             color: ${neutral[60]};
-            border-bottom: 0.0625rem solid ${neutral[60]};
+            border-bottom: none;
```

Comment:
If you leave this unset does a border appear?

Reply:
Nope that can be removed